examples/tutorial3/tutorial/app.py

```diff
@@ -9,18 +9,20 @@ def startup(self):
         self.webview = toga.WebView(style=Pack(flex=1))
         self.url_input = toga.TextInput(
             initial='https://github.com/',
-            style=Pack(flex=1, padding=5)
+            style=Pack(flex=1)
         )
 
         box = toga.Box(
             children=[
                 toga.Box(
                     children=[
                         self.url_input,
-                        toga.Button('Go', on_press=self.load_page, style=Pack(width=50, padding_right=5)),
+                        toga.Button('Go', on_press=self.load_page, style=Pack(width=50, padding_left=5)),
                     ],
                     style=Pack(
```

Comment:
At column 27: (F405) 'Pack' may be undefined, or defined from star imports: toga.style.pack